clarity/src/vm/clarity_wasm.rs

```diff
@@ -5546,7 +5546,12 @@ fn link_print_fn(linker: &mut Linker<ClarityWasmContext>) -> Result<(), Error> {
         .func_wrap(
             "clarity",
             "print",
-            |mut caller: Caller<'_, ClarityWasmContext>, value_offset: i32, value_length: i32| {
+            |mut caller: Caller<'_, ClarityWasmContext>,
+             value_offset: i32,
+             value_length: i32,
+             _serialized_ty_offset: i32,
+             _serialized_ty_length: i32| {
+                // NOTE: _serialized_ty_offset and _serialized_ty_length are used in `stacks-network/clarity-wasm`, here we only care about the function stub
```

Comment:
This isn't completely true. The function here would be called if it's the consequence of a `contract-call?`. So if we want to keep a coherence between functions called directly or from another contract, we have to duplicate it until the duplication issue is solved.

Reply:
I missed that, sorry! Copied all the code (and tested with a contract call, I'll add the test to the other PR).

The only relevant delta is the removal of the dev mode debug from here, as it wasn't present in the clarity-wasm code.